batch_disposition.go

```diff
@@ -16,6 +16,11 @@ type (
 		Status       MessageStatus
 		cursor       int
 	}
+
+	// BatchDispositionErrors returns LockTokenIDs with associated error.
+	BatchDispositionErrors struct {
+		Errors map[*uuid.UUID] error
```

Comment:
What do you think about making this implement error? I think it would be nice if this was just a specialized error.

Reply:
Good idea 👍 ; I'll fix it.